src/_locales/es/messages.json

```diff
@@ -440,47 +440,47 @@
         "description": ""
     },
     "intro_not_an_adblocker_paragraph": {
-        "message": "Privacy Badger comienza a bloquear una vez que ve el mismo rastreador en tres sitios web diferentes. ¡Tres strikes y está afuera! Si usas otras herramientas de privacidad o bloqueadores de anuncios, Privacy Badger aprenderá menos, pero está bien, podrá detectar cosas que tus otros bloqueadores omitan.",
+        "message": "Privacy Badger comienza a bloquear una vez que ve el mismo rastreador en tres sitios web diferentes. ¡Tres strikes y está afuera! Si usa otras herramientas de privacidad o bloqueadores de anuncios, Privacy Badger aprende menos, pero está bien, puede que descubra cosas que no detecten sus otros bloqueadores.",
         "description": "Intro page paragraph."
     },
     "options_domain_status_filter": {
-        "message": "Filtrar por estatus:",
+        "message": "Filtrar por estado:",
         "description": "Label for a dropdown control on the Tracking Domains options page tab."
     },
     "report_text": {
-        "message": "Describe más abajo brevemente el error.",
+        "message": "Describa brevemente el error aquí abajo.",
         "description": ""
     },
     "remove_button": {
-        "message": "Eliminar seleccionado",
+        "message": "Borrar los seleccionados",
         "description": "This is the label for the 'Remove selected' buttons."
     },
     "intro_donate1": {
-        "message": "Cuando utilizas Privacy Badger te unes a <a href='https://www.eff.org/' target='_blank'>Electronic Frontier Foundation</a> y a millones de otros usuarios en la lucha por la privacidad. Somos una organización sin fines de lucro que lucha por tus derechos en línea. ¡Gracias por unirte!",
+        "message": "Cuando utiliza Privacy Badger se une a la <a href='https://www.eff.org/' target='_blank'>Electronic Frontier Foundation</a> y a millones de otros usuarios en la lucha por la privacidad. Somos una organización sin ánimo de lucro que lucha por sus derechos en línea. ¡Gracias por unirse a nosotros!",
         "description": "Part of the 'donate' section on the intro page."
     },
     "options_domain_search_tooltip": {
-        "message": "Para excluir dominios, preceda su término de búsqueda con \"-\". Por ejemplo, \".co -.com\" mostrará dominios .co y .co.uk pero no .com.",
+        "message": "Para excluir dominios, preceda el término de búsqueda con \"-\". Por ejemplo, \".co -.com\" muestra dominios .co y .co.uk, pero no .com.",
```

Comment:
Should we switch the "" quotes here to «» like in the `non_tracker_tip` message?